keras_imaging/datasets/bbbc013.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+
+import keras_imaging.datasets
+
+
+def load_data():
```

Comment:
@sanz999 Write some documentation! You can find an example here:

https://github.com/broadinstitute/keras-rcnn/blob/master/keras_rcnn/datasets/malaria.py#L7-L14